hrjobcontract/js/test/controllers/form/form-pension_test.js

```diff
@@ -0,0 +1,136 @@
+define([
+  'mocks/data/contact',
+  'job-contract/app'
+], function(ContactMock) {
+  'use strict';
+
+  describe('FormPensionCtrl', function() {
+    var ctrl, $controller, $rootScope, $scope, $httpBackend, $q, ContactService, response, params;
+
+    beforeEach(module('hrjc'));
+
+    beforeEach(function() {
+      inject(function(_$controller_, _$rootScope_, _$httpBackend_, _$q_, _ContactService_) {
+        $controller = _$controller_;
+        $rootScope = _$rootScope_;
+        $httpBackend = _$httpBackend_;
+        ContactService = _ContactService_;
+        $q = _$q_;
+        params = {
+          contact_type: 'Organization',
+          contact_sub_type: 'Life_Insurance_Provider'
+        };
+        contactServiceSpy();
+      });
+    });
+
+    beforeEach(function() {
+      $httpBackend.whenGET(/action=get&entity=HRJobContract/).respond({});
+      $httpBackend.whenGET(/views.*/).respond({});
+    });
+
+    describe('On initialization', function() {
+      describe('Case I: when $scope.entity.pension.pension_type is truthy', function() {
+        beforeEach(function() {
+          initController(1);
+          $scope.$digest();
+        });
+
+        it('defines contact object', function() {
+          expect($scope.contacts).toBeDefined();
+        });
+
+        it("calls getOne()", function() {
+          expect(ContactService.getOne).toHaveBeenCalled();
+        });
+
+        it("sets value for $scope.contacts.Pension_Provider", function() {
```

Comment:
in test cases describe the variables and ps avoid using them directly so this could be 

> sets the values for contact's pension provider